mlops/k8s_templates/instance-type.yaml

```diff
@@ -3,12 +3,10 @@ kind: InstanceType
 metadata:
   name: computeinstancetype
 spec:
-  nodeSelector:
-    agentpool: confcompool
```

Comment:
Doesn't that make it not a conf comp anymore? If so, wouldn't that be a problem with `silo_open_aks_with_cc.md`?

Reply:
Not at all, adding `confcomppool` selector just limits us picking compute from this pool. However, with regular aks you can possibly name your node pool differently. The most generic way how to implement instance-type is to let the script choose most suitable compute out of all of your available pools in the cluster.